scripts/qesap/lib/cmds.py

```diff
@@ -58,6 +58,8 @@ def create_hana_media(config_ansible, apiver):
     hanamedia_content['az_container_name'] = config_ansible['az_container_name']
     if 'az_sas_token' in config_ansible:
         hanamedia_content['az_sas_token'] = config_ansible['az_sas_token']
+    if 'az_key_name' in config_ansible:
+        hanamedia_content['az_key_name'] = config_ansible['az_key_name']
```

Comment:
This could be an `elif`, but maybe it's better this way - conflicts will be handled in the playbook to help with manual deployments too

Reply:
No, I intentionally place `if`. I like the final `hana_media.yaml` to potentially has both, and at least one.

This way I think Ansible cold have all the freedom deciding what to do:
Ansible can (my preferred) if `az_sas_token` is provided, just use it and ignore `az_key_name`. If token is not provided calculate it on the fly been sure that needed argument `az_key_name` is in available as input.